v19.1/recommended-production-settings.md

```diff
@@ -43,25 +43,35 @@ For added context about CockroachDB's fault tolerance and automated repair capab
 
 ## Hardware
 
+### Terminology
+
+To properly select your cluster's hardware, it's important to review define basic hardware terminology:
+
+Term | Definition
+-----|------------
+**CPU** | The [physical central processing unit (CPU)](https://en.wikipedia.org/wiki/Central_processing_unit) that runs a CockroachDB server run on a (virtual) machine.</br>Also known as: central processor, main processor
+**vCPU** | Four virtual central processing units (vCPU) comprise a CPU. vCPUs increase a core's performance by running multiple tasks simultaneously. </br>Also known as: hyperthread, scheduling unit
```

Comment:
A CPU can have any number of vCPUs, not always 4. 

Hyperthreading increases a core's performance relative to non-hyperthreaded cores, although whether there is hyperthreading or not there are still vCPUs. It's just a question of whether there is more than one vCPU per core.